lib/liquid/template.rb

```diff
@@ -78,17 +76,16 @@ def taint_mode=(mode)
         @taint_mode = mode
       end
 
+      # Sets the file system.
+      attr_writer :file_system
```

Comment:
Why didn't you define this next to the reader method?

Reply:
Simply because other writers were at the top and readers at the bottom. 

No strong opinion here.